### PR TITLE
Update Windows requirement from Vista (6.0) to 7 (6.1).

### PR DESCRIPTION
**Pull request by jaykrell**
Still to see what _WIN32_IE should be, maybe omit it.

Part of #9102

### CODE REVIEW

**Comment by jaykrell** [comment]
arm64  roslyn failure

**Comment by jaykrell** [comment]
@monojenkins build  failed

**Comment by jaykrell** [comment]
@monojenkins build failed

File: configure.ac [hunk 1]
```diff
@@ -145,7 +145,7 @@ case "$host" in
 		fi
 		HOST_CC="gcc"
 		# Windows Vista or later is required
```
Comment (prajaybasu) [inline configure.ac]: Can this comment be updated to reflect the changes?

Comment (jaykrell) [reply configure.ac]: Yes good catch. I should have grepped for Vista. (and maybe  say "Windows7" instead of "Windows 7" for future greps)

**Comment by akoeplinger** [comment]
There's another place that should be changed:
https://github.com/mono/mono/blob/885a7aaf4e5412339e528fefbba02b6c279d5787/packaging/Windows/resources/Product.wxs#L42-L44

**Comment by jaykrell** [comment]
Good catch. I'll update both of these later today. And grep a bit more. Approved otherwise?

**Comment by jaykrell** [comment]
This file could be further trimmed but maybe a later PR?

mcs/class/System.Security/System.Security.Cryptography/ProtectedMemory.cs

**Comment by jaykrell** [comment]
I'll add it here and dismiss review so rereview.

**Comment by jaykrell** [comment]
mcs/class/referencesource/System.ServiceModel/System/ServiceModel/Security/SecurityUtils.cs:        internal static bool IsOSGreaterThanOrEqualToWin7()

mcs/class/referencesource/System.ServiceModel/System/ServiceModel/Security/WindowsSspiNegotiation.cs:                    && SecurityUtils.IsOSGreaterThanOrEqualToWin7()


jaymac1:mono2 jay$ git grep IsOsGreaterThanXP
mcs/class/referencesource/System.ServiceModel/System/ServiceModel/Security/SecurityUtils.cs:        internal static bool IsOsGreaterThanXP()
mcs/class/referencesource/System.ServiceModel/System/ServiceModel/Security/SecurityUtils.cs:                if (SecurityUtils.IsOsGreaterThanXP())
mcs/class/referencesource/System.ServiceModel/System/ServiceModel/Security/SpnegoTokenProvider.cs:            bool osIsGreaterThanXP = SecurityUtils.IsOsGreaterThanXP();
mcs/class/referencesource/System.ServiceModel/System/ServiceModel/Security/SpnegoTokenProvider.cs:            if (!this.allowNtlm && !SecurityUtils.IsOsGreaterThanXP())


What is procedure for updating referencesource?
Hope for it to be deleted and replaced by corefx?

**Comment by jaykrell** [comment]
Last change caused problems, will undo, leaving just PR commented changes.

**Comment by jaykrell** [comment]
Now should represent what was approved + PR feedback. The additional Win2000 etc. support put back.

**Pull request merged**